From 838b865b395c2fa189bc01027d606659aa7df169 Mon Sep 17 00:00:00 2001
From: Ian Chamberlain <ian.h.chamberlain@gmail.com>
Date: Sun, 19 Mar 2023 12:43:50 -0400
Subject: [PATCH] Allow older kernel version for %p4cc format

SteamOS kernel has backported this feature to 5.13 kernel, so it will
still build even if the kernel is older than 5.14.
---
 module/evdi_painter.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/module/evdi_painter.c b/module/evdi_painter.c
index eff8395..5e1c8b7 100644
--- a/module/evdi_painter.c
+++ b/module/evdi_painter.c
@@ -725,11 +725,11 @@ void evdi_painter_dpms_notify(struct evdi_device *evdi, int mode)
 }
 
 static void evdi_log_pixel_format(uint32_t pixel_format,
 		char *buf, size_t size)
 {
-#if KERNEL_VERSION(5, 14, 0) <= LINUX_VERSION_CODE
+#if KERNEL_VERSION(5, 13, 0) <= LINUX_VERSION_CODE
 	snprintf(buf, size, "pixel format %p4cc", &pixel_format);
 #else
 	struct drm_format_name_buf format_name;
 
 	drm_get_format_name(pixel_format, &format_name);
-- 
2.39.0

